source/technical/metrics.rst

```diff
@@ -229,3 +229,74 @@ Note the following caveats:
 - Monitoring periods larger than 10-15 minutes may be inaccurate due to RAPL registers overflowing if they're not read regularly.
 
 This `utility <https://github.com/coolr-hpc/intercoolr>`_  was contributed by Chameleon user `Kazutomo Yoshii <http://www.mcs.anl.gov/person/kazutomo-yoshii>`_ of `Argonne National Laboratory <http://www.anl.gov/>`_.
+
+_____________________________________________
+Power Consumption Metrics for Low Power Nodes
+_____________________________________________
+
+In addition to the system and power consumption metrics described above, Chameleon automatically collects power usage data on all low power nodes in the system. Instantaneous power usage data (in watts) are collected through the IPMI interface on the chassis controller for the nodes. This “out-of-band” approach does not consume additional power on the node itself and runs even when the node is powered off. Low power nodes for which power usage data are now being collected include all Intel Atoms, low power Xeons, and ARM64s.
+
+As with the system metrics, retrieving the power consumption metrics for a low power node requires the OpenStack CLI and Gnocchi client plugin (see installation instructions above). Retrieve the power usage metrics using the following command:
+
+.. code-block:: bash
+
+   $ openstack metric measures show power --resource-id=<node_uuid> --refresh
+
+
+.. tip::
+   The node UUID and the instance UUID are different. You can get a node's UUID for a reservation from the Horizon GUI (https://chi.tacc.chameleoncloud.org for TACC reservations, https://chi.uc.chameleoncloud.org for UC reservations). Click on your lease name from within the list of leases on the Leases subtab within the Reservations tab. The node UUID is at the very bottom under the ``Nodes`` section.
```

Comment:
Now that we have multi-region, we can remove the part between parenthesis: TACC reservations are accessible from UC's Horizon, and vice versa.

Reply:
"Click on your lease name from within the list of leases on the Leases subtab within the Reservations tab.": that's very verbose, I think any user who reached this part of the tutorial would already know how to access the lease tab.